src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  margin: 0;
+  display: block;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  display: block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+nav {
+  display: block;
+  unicode-bidi: isolate;
+}
+
+li {
+  display: list-item;
+  text-align: -webkit-match-parent;
+  unicode-bidi: isolate;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling. You should style the `.nav__list` using class selectors without including the `ul` tag name as part of the selector.